### DISCUSSION TITLE
posts/2020/08/google-fonts-privacy/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#18] [Announcements]
# posts/2020/08/google-fonts-privacy/

Explaining changes I’ve made to this site because of its new privacy policy.

https://www.brycewray.com/posts/2020/08/google-fonts-privacy/

**Comment by ryanjsfx2424** [comment]
Interesting on ways to avoid running afoul of the GDPR. It seems rather ridiculous that they'd go after individuals possibly unaware of google fonts or other embedded content (same issue with bootstrap?) having this issue as opposed to Google or the large corporations itself. That said, the GDPR also feels rather infantilizing; people that really care about privacy can use TOR and/or browsers with protective measures (I like Brave and I think Firefox is starting to bake that in). I hat the "accept" cookie boxes and figure most people don't care (like I don't; most of the time I don't use TOR).

**Reply by brycewray** [reply]
I would agree that most people ignore all the cookies warnings and that the tech-savvy can protect themselves in other ways. However, we must always keep in mind that the vast majority of web users are **not** tech-savvy and don’t know the first thing about either how to protect themselves or why they should. As with all regulatory mechanisms, the intent is to protect the weakest lambs among the flock; the big, bad, butt-kicking-and-name-taking rams can always take care of themselves.
